src/main/scala/com/codacy/api/helpers/vcs/GitClient.scala

```diff
@@ -22,4 +25,16 @@ class GitClient(workDirectory: File) {
     }.toOption.flatten.filter(_.trim.nonEmpty)
   }
 
+  def latestCommitInfo: Option[CommitInfo] = {
+    Try {
+      repository.map { rep =>
+        val git = new Git(rep)
+        val headRev = git.log().setMaxCount(1).call().asScala.head
+        val authorIdent = headRev.getAuthorIdent
+
+        CommitInfo(headRev.getName, authorIdent.getName, authorIdent.getEmailAddress, authorIdent.getWhen)
+      }
+    }.toOption.flatten
```

Comment:
I know this lib has the .toOption everywhere, but could we return the try instead?